packages/contracts/hardhat.config.ts

```diff
@@ -43,7 +43,7 @@ const config: HardhatUserConfig = {
     timeout: 50000,
   },
   solidity: {
-    version: '0.7.6',
+    version: '0.8.4',
```

Comment:
Can't we use the latest - 0.8.7?

Reply:
hardhat doesn't support 0.8.7 yet, so i figured we could update when it's supported since the main lift is upgrading to 0.8!

Reply:
Do we use any hardhat features that aren't supported yet in 0.8.7? I'd imagine things like `hardhat compile` still work, its just extra stuff like stack traces from my understanding

Reply:
gotcha, if we don't use the extra features then there was no other reason for not using 0.8.7